container-chains/templates/frontier/node/src/command.rs

```diff
@@ -225,21 +245,22 @@ pub fn run() -> Result<()> {
                 cmd.run(&*spec)
             })
         }
+        Some(Subcommand::FrontierDb(_)) => todo!(),
```

Comment:
This command is likely to be deprecated. The original idea was to be able to explore the secondary rocks db using the cli, but we are probably moving to sql now that is merged upstream, and we obviously don't need this for it.